src/frontends/onnx/frontend/src/op/com.microsoft/attention.cpp

```diff
@@ -153,7 +153,7 @@ ov::NodeVector split_to_QKV(const std::shared_ptr<v1::Add>& node,
         // head_size = hidden_size / num_heads
         head_size = std::make_shared<v1::Divide>(hidden_size, num_heads_node);
         // split the node into 3 even parts Q, K, V with shape (batch_size, sequence_len, hidden_size)
-        split = ov::op::util::split(node, 3, 2);
+        split = ov::op::util::make_split(node, 3, 2);
```

Comment:
as I see, the code from test utils is still used in the production code. Can we avoid it?

Reply:
@ilya-lavrenov @vurusovs 
This is not a test utils. This is a separated cmake target. But it is located in src/tests/...
I can move this target from src/tests/ov_helpers/ov_models/ov_builders to dev API.
Do you have objections to this approach?

Reply:
As we discussed, there should be no CMake target for production builders, it should be fully moved to ONNX FE (if duplications required, let's duplicate it), and no includes from tests in production code should be presented

Reply:
They are unfortunately used not only in ONNX frontend. So we cannot do it

Reply:
Let's make copy of them, the list should be very short

Reply:
Yes, it may take a place

Reply:
done. please re-review